docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -24,7 +24,11 @@ pre = "<b> </b>"
 ---
 ### Overview
 
-Use configuration overrides (also called _situational configuration_) to customize a WebLogic domain home configuration without modifying the domain's actual `config.xml` or system resource files. For example, you may want to override a JDBC data source XML module user name, password, and URL so that it references a local database.
+{{% notice note %}}
+Configuration overrides can only be used in combination with Domain in Image and Domain in PV domains. For Model in Image domains, use [Model in Image Runtime Updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}) instead.
+{{% /notice %}}
+
+Use configuration overrides (also called _situational configuration_) to customize a Model in Image or Domain in PV domain's WebLogic domain home configuration without modifying the domain's actual `config.xml` or system resource files. For example, you may want to override a JDBC data source XML module user name, password, and URL so that it references a local database.
```

Comment:
I thought config overrides are not supported for Model in Image. Typo? "Model in Image" -> "Domain in Image"?

Reply:
Oops! Yes that's a typo! I'll fix this and resolve once fixed.

Reply:
Fixed.